app/assets/stylesheets/themes/cyborg.css

```diff
@@ -0,0 +1,71 @@
+/* TODO: debug why _head.html+cyborg.erb isn't loading fonts right */
```

Comment:
What's the issue here?

Reply:
This was before I found _typography.scss :). I'll remove the extraneous comment.